unit/api/application_resource.py

```diff
@@ -22,8 +22,18 @@ def create(self, request: Union[CreateIndividualApplicationRequest, CreateBusine
         else:
             return UnitError.from_json_api(response.json())
 
-    def list(self, offset: int = 0, limit: int = 100) -> Union[UnitResponse[List[ApplicationDTO]], UnitError]:
-        response = super().get(self.resource, {"page[limit]": limit, "page[offset]": offset})
+    def list(self, params: ListApplicationParams = None) -> Union[UnitResponse[List[ApplicationDTO]], UnitError]:
+        params = params or ListApplicationParams()
```

Comment:
Hey @axshani, I noticed `ListAccountParams` is being constructed in the function signature [here](https://github.com/unit-finance/unit-python-sdk/blob/5cb40775758446306b55f2f6a2ced432a392d5eb/unit/api/account_resource.py#L50) (possibly other places as well). You may want to consider updating to how I have it here (set the default to `None`) since `None` is an immutable datatype in python. 

Little document to explain why [here](https://docs.python-guide.org/writing/gotchas/)